user-guide/Basic_Functionality/Protocols_and_templates/Alarm_templates/Configuring_alarm_templates/Configuring_anomaly_detection_alarms.md

```diff
@@ -4,7 +4,74 @@ uid: Configuring_anomaly_detection_alarms
 
 # Configuring anomaly detection alarms for specific parameters
 
-From DataMiner 10.0.3 onwards, you can enable alarm monitoring on specific types of anomalies for parameters in an alarm template. If you enable this, an alarm is generated whenever an anomaly of the relevant type is detected for those parameters.
+From DataMiner 10.0.3 onwards, you can enable alarm monitoring on specific types of anomalies for parameters in an alarm template. If you enable this, an alarm is generated whenever an anomaly of the relevant type is detected for those parameters. From DataMiner 10.2.0 [CU21]/10.3.0 [CU9]/10.3.12 onwards<!--RN 37171-->, you can choose whether an alarm or a suggestion event is generated for a specific type of anomaly detection, e.g. have suggestions appear for downward level shifts while alarms occur for upward level shifts.
+
+> [!NOTE]
+> Suggestion events are cleared two hours after their creation time or their last update time. You can view them by creating a suggestion event tab in the Alarm Console. See [Adding and removing alarm tabs in the Alarm Console](xref:ChangingTheAlarmConsoleLayout#adding-and-removing-alarm-tabs-in-the-alarm-console).
+
+## [From DataMiner 10.2.0 [CU21]/10.3.0 [CU9]/10.3.12 onwards](#tab/tabid-1)
+
+To configure anomaly detection alarms for specific parameters<!--RN 37171 + 37148-->:
+
+1. In the alarm template editor, click the *Anomalies* button for the parameter in question. By default, this is set to *Disabled*.
+
+1. In the *Anomaly Alarm Settings* pop-up window, you can select one of the preset options or configure the anomaly alarm settings according to your preference.
+
+   > [!NOTE]
+   > The configured anomaly alarm settings only take effect for trended parameters<!--RN 37670-->.
+
+   - To choose one of the preset options, click the *Select preset* selection box in the top-left corner:
+
+     - **All disabled**: No anomaly alarms are generated.
+
+     - **All smart**: Anomaly alarms are generated based on what DataMiner Analytics determines to be an anomaly.
+
+   - To customize the anomaly alarm settings:
+
+     1. Configure alarms for specific types of anomaly detection:
+
+        - *Level shift > Level increases*: Enables or disables alarms for level increase anomalies.
+
+        - *Level shift > Level decreases*: Enables or disables alarms for level decrease anomalies.
+
+        - *Outlier > Upward spikes*: Enables or disables alarms for upward spikes.
+
+        - *Outlier > Downward spikes*: Enables or disables alarms for downward spikes.
```

Comment:
I think you should also include some explanation about the custom thresholds that can be set for level shifts and outliers:

- You can set absolute thresholds by selecting *Absolute* in the drop down menu next to each change point type: this will alarm level shifts based on the size of the jump (resp. outliers based on how far the outlier is from the surrounding points in absolute numbers). e.g. if you select *Absolute* for level increases and fill in 150 in the box for severity major, then a parameter jumping from, say, 100 to 255 will get a major alarm (since 255 - 100 = 155 > 150). If it jumped from 100 to 245 on the other hand, no alarm (and also no suggestion event) is generated. 
- You can set relative thresholds by selecting *Relative* in the drop down menu next to each change point type: this will alarm level shifts based on the size of the jump *relative to the value where the jump started*. This is then expressed in %. E.g. in the same example as above, if we select relative (instead of absolute) for level increases and set a value of 100% in the box for severity major, then the jump from 100 to 255 (i.e. 155% increase) or from 100 to 245 (i.e. 145% increase) will both get major alarms, but a jump from 200 to 250 will not get an alarm, since that is only an increase with 25%.